foundationsync/indico_foundationsync.py

```diff
@@ -80,6 +84,10 @@ def connect_to_foundation(self):
     def _get_room_attrs(self, raw_data):
         return {'manager-group': raw_data.get('EMAIL_LIST')}
 
+    def _fix(self, s):
```

Comment:
`_fix` seems like too broad of a name. What do you think of `_fix_html`? Or event `_html_to_markdown`, since that's what the function actually does.